client/webserver/site/src/js/markets.ts

```diff
@@ -1749,6 +1796,90 @@ export default class MarketsPage extends BasePage {
     }
   }
 
+  /*
+   * ordersSortCompare returns sort compare function according to the active
+   * sort key and direction.
+   */
+  recentMatchesSortCompare () {
+    console.log(this.recentMatchesSortKey)
+    switch (this.recentMatchesSortKey) {
+      case 'rate':
+        return (a: BasicMatches, b: BasicMatches) => this.recentMatchesSortDirection * (a.rate - b.rate)
+      case 'qty':
+        return (a: BasicMatches, b: BasicMatches) => this.recentMatchesSortDirection * (a.qty - b.qty)
+      case 'stamp':
+        return (a: BasicMatches, b: BasicMatches) => this.recentMatchesSortDirection * (b.age.getTime() - a.age.getTime())
+
+    // Sort by taker and maker?
```

Comment:
Does sorting by taker and maker make sense?

Reply:
My suggestion to include the OIDs was really just for the server->client messaging, which gives the backend some more capability to audit the matches based on the match_proof and other orderbook subscription data.  The frontend can be very simple.